letsencrypt/client/client.py

```diff
@@ -23,6 +28,28 @@
 from letsencrypt.client.display import enhancements
 
 
+SETUPTOOLS_AUTHENTICATORS_ENTRY_POINT = "letsencrypt.authenticators"
+"""Setuptools entry point group name for Authenticator plugins."""
+
+
+def init_auths(config):
```

Comment:
Exactly, I wasn't sure myself where to put this code. I'll move it to `main.py` for now, but we could think about something like `client/cli.py`...